doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+* Why Spacemacs?
+- Unparallelled text and structure editing for all types of writing tasks: creative writing, blogging, note-taking, todo-lists,
+  scientific papers... 
+- Powerful modes for programming in dozens of programming languages
+- Deeply customizable yet beginner-friendly
+* Installation
+ Spacemacs is a beginner-friendly and powerful distribution of a popular text
+ editor called Emacs. To install Spacemacs you need to first install base Emacs and then download
+ and install the Spacemacs extension files, which is easiest done by using a program
+ called Git. The steps are easy and outlined below.
+** 1. Install Emacs
+Follow the instructions in https://github.com/syl20bnr/spacemacs#prerequisites.
+** 2. Install Git
+Go to https://git-scm.com/downloads and install Git on your platform.
+** 3. Install Spacemacs
+Open a terminal or command prompt, paste the following code to it:
+
+git clone https://github.com/syl20bnr/spacemacs ~/.emacs.d
+
+Press enter to execute the code and the program you installed in the previous step,
+Git, will download the Spacemacs extension files.
+** 4. Install the default font
+The default font used by Spacemacs is Source Code Pro by Adobe. It is
+recommended to [[https://github.com/adobe-fonts/source-code-pro][install]] it on your system to ensure correct visual representation.
+** Done!
+Open Spacemacs by clicking the Emacs icon in your applications menu. The first
+time Spacemacs launches, it will ask you for your preferred editing style. We
+recommend selecting the default option, "Evil", more of which is talked about in
```

Comment:
I would recommend either `evil` or `emacs` based on user background. Emacs users trying Spacemacs for the first time without any vim background should not use `evil` unless they are here for `evil` part. 

I think the main point I would like to make here is that user might choose any editing style based on his preference and background. `hybrid` style is generally for advanced users. **But** it's very important to say that any decisions made here can be easily changed later on.